src/styles/patterns/dswp-horizontal-card-large-img-right-no-shadow.scss

```diff
@@ -0,0 +1,40 @@
+@media (max-width: $break-medium) {
+
+	.dswp-horizontal-card-large-img-no-shadow {
+
+		img {
+			width: 100% !important;
```

Comment:
i'm guessing we need the important on these styles because we are trying to fight WordPress default styles?

Reply:
correct